charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -59,6 +59,11 @@ securityContext: {}
 config:
   datasource:
     connectionSecret:
+      # we have a problem here...
```

Comment:
this is something we need to discuss and come up with whatever solution we want to use.  i haven't figured out anything very good.

i got it working by modifying the `astronomer-bootstrap` secret value into a new temporary one to make it work for testing purposes, but that isn't going to work going forward...

Reply:
Need to have a think on this one

Reply:
at the easiest we could add a blank database called `airflow` i guess.... but i am unsure if not having that database was done on purpose or not

Reply:
Getting in to hack territory, but what about an init container on the exporter that reads in the `astronomer-houston-backend` secret, and mutates it to what the exporter needs it to look like, then creates a secret with that value?

Reply:
I don't like how this feature turned out when it was made in the first, place so sorry about that. I was having issues with consistently populating the configuration, considering for connection string variations or the other way of providing connection when using SSL. This is another part of the puzzle https://github.com/astronomer/ap-vendor/blob/main/postgres-exporter/include/entrypoint